src/operator/controllers/monitor.go

```diff
@@ -646,114 +644,6 @@ func (m *VizierMonitor) repairVizier(state *vizierState) error {
 		}
 
 		log.Info("Successfully switched to etcd backed metadata store")
-	} else if state.Reason == status.TLSCertsExpired {
-		vz := &pixiev1alpha1.Vizier{}
-		err := m.vzGet(context.Background(), m.namespacedName, vz)
-		if err != nil {
-			log.WithError(err).Error("Failed to get vizier")
-			return err
-		}
-
-		err = deployCerts(context.Background(), m.namespace, vz, m.clientset, m.restConfig, true)
-		if err != nil {
-			log.WithError(err).Error("Failed to update certs")
-		}
-		m.certState = okState()
-
-		log.Info("Bouncing Vizier pods to get certs update")
-		err = k8s.DeletePods(m.clientset, m.namespace, "")
-		if err != nil {
-			return err
-		}
```

Comment:
we should keep this section right?

Reply:
Whoops, yes, accidentally removed it. Thanks for catching!